doobie-hikari/src/main/scala/com/avast/sst/doobie/DoobieHikariModule.scala

```diff
@@ -0,0 +1,44 @@
+package com.avast.sst.doobie
+
+import cats.effect.{Async, Blocker, ContextShift, Resource, Sync}
+import com.zaxxer.hikari.HikariConfig
+import com.zaxxer.hikari.metrics.MetricsTrackerFactory
+import doobie.hikari.HikariTransactor
+
+import scala.concurrent.ExecutionContext
+
+object DoobieHikariModule {
+
+  /** Makes [[doobie.hikari.HikariTransactor]] initialized with the given config.
+    *
+    * @param boundedConnectExecutionContext [[scala.concurrent.ExecutionContext]] used for creating connections (should be bounded!)
+    */
+  def make[F[_]: Async](config: DoobieHikariConfig,
+                        boundedConnectExecutionContext: ExecutionContext,
+                        blocker: Blocker,
+                        metricsTrackerFactory: MetricsTrackerFactory)(implicit cs: ContextShift[F]): Resource[F, HikariTransactor[F]] = {
```

Comment:
to make things general, let's make `MetricsTrackerFactory` optional, since it doesn't have to be set

Reply:
Right.